### PR TITLE
Compatibility with latest nearcore stable branch

### PR DESCRIPTION
**Pull request by emaG3m**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ const NEAR_NET_VERSION_TEST = '98';
 const utils = require('./utils');
 const nearToEth = require('./near_to_eth_objects');
 
-const GAS_AMOUNT = new BN('1000000000000000'); // 10e14.
+const GAS_AMOUNT = new BN('200000000000000');
```
Comment (barbaraliau) [inline src/index.js:12]: What was the error for this one? We've had to change the `GAS_AMOUNT` several times lol.

File: test/near_provider.test.js [hunk 2]
```diff
@@ -617,12 +617,13 @@ describe('\n---- PROVIDER ----', () => {
...
 
             test('errors if not a real txhash', withWeb3(async (web) => {
+                const errorType = "[-32602]";
                 try {
                     const badHash = 'whatsuppppp:hello';
                     await web.eth.getTransactionReceipt(badHash);
                 } catch (e) {
                     expect(e).toBeTruthy();
-                    expect(e.message).toEqual('[-32602] Invalid params: Failed parsing args: incorrect length for hash');
+                    expect(e.message).toContain(errorType);
```
Comment (barbaraliau) [inline test/near_provider.test.js:626]: Nice. I forgot about `toContain`.

File: src/index.js (hunk 1 above)
Comment (emaG3m) [reply src/index.js:12]:
```
InvalidTxError: {}
          at Object.parseRpcError (/Users/emilywilliams/code/summa/near/near-web3-provider/node_modules/near-api-js/lib/utils/rpc_errors.js:38:19)
          at JsonRpcProvider.sendJsonRpc (/Users/emilywilliams/code/summa/near/near-web3-provider/node_modules/near-api-js/lib/providers/json-rpc-provider.js:120:40)
          at processTicksAndRejections (internal/process/task_queues.js:97:5)
          at Account.signAndSendTransaction (/Users/emilywilliams/code/summa/near/near-web3-provider/node_modules/near-api-js/lib/account.js:97:22)
          at NearProvider.routeEthSendTransaction (/Users/emilywilliams/code/summa/near/near-web3-provider/src/index.js:714:27) {
        type: 'InvalidTxError'
      }
```

File: test/near_provider.test.js (hunk 2 above)
Comment (emaG3m) [reply test/near_provider.test.js:626]: we get a different and much more confusing new error message for this now hah

File: src/index.js (hunk 1 above)
Comment (emaG3m) [reply src/index.js:12]: not very helpful --^

**Pull request merged**